packages/common-scripts/src/locktime_pool.ts

```diff
@@ -264,7 +263,8 @@ export const CellCollector: CellCollectorConstructor = class CellCollector
             !validateSince(
               since!,
               this.tipSinceValidationInfo!,
-              sinceValidationInfo
+              // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
```

Comment:
if we narrow the `sinceValidationInfo` type as 
```
if (
          parseSinceCompatible(since!).type === "blockTimestamp" ||
          (this.tipHeader &&
            && sinceValidationInfo &&
            !validateSince(
              since!,
              this.tipSinceValidationInfo!,
              sinceValidationInfo!
            ))
```
The unit test will throw an error. Seems non-null-type-assertion is necessary here